aleksandra/functions/homework.js

```diff
@@ -0,0 +1,79 @@
+// This is homework from 17-11-2022 JS class. We learned about functions. 
+
+/*
+The task was to: 
+Create functions for all the learned arithmetic operations, 
+and use them in the file, console.log results. 
+Every function needs to accept 2 params.
++    
+-    
+/    
+*    
+**    
+%
+*/
+
+//The basic step is to define the two numbers that we will use in the functions. 
+//We will do this by assigning them to variables. 
+//I used 'let' because the numbers can be different. Also I could use 'var'. 
+//*The difference between 'let' and 'var' we will learn during this course. 
+
+let a;
+let b; 
```

Comment:
You should not declare these variables here, they're not used anywhere, and might lead to some confusion in VSCode.
There is a difference between a variable and function parameter. the:
```
let a;
```
is not used in
```
function (a) {}
```
but instead that `a` from function becomes inner scope of function declaration, and `a` above is..,. well, above it, as a declared function.